.github/workflows/continuous-documentation.yml

```diff
@@ -83,9 +83,15 @@ jobs:
       - name: Deploy documentation to Vercel
         uses: amondnet/vercel-action@v20.0.0
         with:
-          # https://vercel.com/account/tokens
+          # Generate a vercel token at https://vercel.com/account/tokens,
+          # and save the value in the secrets setting in the repository.
           vercel-token: ${{ secrets.VERCEL_TOKEN }}
-          # vercel-org-id and vercel-project-id are generated using vercel CLI
+          # https://github.com/marketplace/actions/vercel-action#project-linking
+          # Install the Vercel CLI and run `vercel` in the project directory.
+          # After linking the repository with a Vercel project, `vercel` will
+          # create a `.vercel` directory containing both the organization (vercel-org-id)
```

Comment:
I didn't checkout this branch locally, but is the a `.vercel` directory we need to add to `.gitignore`?

Reply:
Running the `vercel` command will create the `.vercel` directory, with a `project.json` file inside. The `project.json` file contains the id information:
```
{"orgId":"example_org_id","projectId":"example_project_id"}
```

The directory and the file are required if I want to deploy the documentation (maybe using a command like `vercel deploy`) from my own computer. But for GitHub Actions, we only need to provide the ids and the action should run the command for us. So, `.vercel` are not added to the repository, and I also deleted the `.vercel` directory after adding the IDs to secrets.

Reply:
Ok, no need to add to .gitignore then.